internal/oci/runtime_vm.go

```diff
@@ -670,6 +670,27 @@ func (r *runtimeVM) StopContainer(ctx context.Context, c *Container, timeout int
 	return nil
 }
 
+// shouldBeStopped checks whether the container's state permits
+// stopping. It determines if stopping the container makes sense
+// based on its current state. A container cannot be stopped if
+// it is already stopped or paused. If the container is paused,
+// the function attempts to unpause it and update its status.
+func (r *runtimeVM) shouldBeStopped(ctx context.Context, c *Container) error {
+	switch c.State().Status {
+	case ContainerStateStopped: // no-op
```

Comment:
Returning an error is not a no-op. Skipping it or allowing a fallthrough would be.

If this needs to be treated as such, then it might warrant a better comment.